__tests__/components/button.test.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import renderer from "react-test-renderer";
+
+describe('Button test',function(){
+  var Button = require('../../src/components/button').default
+  var input = [{
+    isActive: '1',
+    theme: 'white05'
+  }];
+
+  it('Button renders when active', () => {
+    var button = renderer.create(<Button
+      params = {input}
+    />).toJSON();
+    expect(button).toMatchSnapshot();
+  });
+})
```

Comment:
Interesting, prettier should fix that. Before you commit. I wonder why it didn't work for you.